src/Routes/Groups/Groups.js

```diff
@@ -12,14 +12,16 @@ import CreateGroupModal from './CreateGroupModal';
 import RenameGroupModal from './RenameGroupModal';
 import DeleteGroupModal from './DeleteGroupModal';
 import useApi from '../../hooks/useApi';
-import { useLocation } from 'react-router-dom';
 import { emptyStateNoFilters, useFeatureFlags } from '../../utils';
+import PropTypes from 'prop-types';
+import Images from '../ImageManager/Images';
 
-const Groups = () => {
+// eslint-disable-next-line react/prop-types
+const Groups = ({ locationProp, navigateProp }) => {
   const hideCreateGroupsEnabled = useFeatureFlags(
     'edge-management.hide-create-group'
   );
-  const { search } = useLocation();
+
```

Comment:
instead of removing maybe you should do like here https://github.com/RedHatInsights/edge-frontend/blob/master/src/Routes/Devices/DeviceTable.js#L272

Reply:
```js
const { search } = locationProp
    ? locationProp()
    : useLocation
    ? useLocation()
    : null ;
```